styles/AdvancedEditor.css

```diff
@@ -550,9 +550,13 @@ Next levels are shifted 'up' by 1
 }
 
 .keywords h1 {
+  font-size: 12px;
+}
+
+.keywords h1:before {
+  content: 'Keywords';
```

Comment:
Unless that is a placeholder, I would suggest to avoid putting text into css. Ideally a nodeview should be in charge of adding html elements that are not a part of PMNode to the view